bin/startup.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
 # Setting Mellon key, cert and metadata files
-if grep -q "libapache2-mod-auth-mellon" "${HOME}/Aptfile" ; then
+if grep -q "auth-mellon" "${HOME}/.apache-mods" ; then
   
+  export APACHE_WORKER_SIZE="${APACHE_WORKER_SIZE:="10"}"
```

Comment:
Why only if mellon is present? It should be all the time no?

Reply:
Mistake - fixed